cirun.yml

```diff
@@ -0,0 +1,22 @@
+# Self-Hosted Github Action Runners on GCP via Cirun.io
```

Comment:
Random question - does this file need to exist at the root of the repo? Also does it have to be visible, as opposed to say `.cirun.yml`?

Reply:
> does this file need to exist at the root of the repo? 

Yes, at the moment Cirun doesn't have the ability to scan the whole repo for config.

> Also does it have to be visible, as opposed to say .cirun.yml?

No, I can make it invisible too, that makes more sense indeed.

Reply:
> Yes, at the moment Cirun doesn't have the ability to scan the whole repo for config.

Even in `.github/cirun.yml` would be great and make sense.

But it's not a big deal, just curious :)

Reply:
> Even in .github/cirun.yml would be great and make sense.

Makes sense, I initially considered it, but was worried in case it conflicts with anything github related, but now looking at the docs I think it shouldn't. I'll incorporate this soon, but cannot promise in this PR.